lib/grp.gd

```diff
@@ -592,6 +592,7 @@ DeclareProperty( "IsPerfectGroup", IsGroup );
 InstallFactorMaintenance( IsPerfectGroup,
     IsGroup and IsPerfectGroup, IsObject, IsGroup );
 
+InstallTrueMethod( IsPerfectGroup, IsGroup and IsTrivial );
```

Comment:
In this case, you could also compensate for the now missing "other" direction, via
`InstallTrueMethod( HasIsPerfectGroup, IsGroup and IsSolvableGroup and IsNonTrivial );`

Reply:
OK